pkg/api/resource_types.go

```diff
@@ -1,8 +1,10 @@
 package api
 
 import (
+	"encoding/json"
 	"strconv"
 
+	"github.com/openshift-online/maestro/pkg/errors"
```

Comment:
could you follow the standard for the order of imports? I will introduce `gci` soon to format the orders.

Reply:
done.